rule_packages/c/IO4.json

```diff
@@ -0,0 +1,74 @@
+{
+  "CERT-C": {
+    "FIO45-C": {
+      "properties": {
+        "obligation": "rule"
+      },
+      "queries": [
+        {
+          "description": "Avoid TOCTOU race conditions accessing files",
```

Comment:
possible improvement to descriptions in this file - add the "causes behavior Y" part of a description. example "TOCTOU race conditions when accessing files can lead to vulnerability."